backend/Wheelingful.BLL/Models/Responses/FetchChapterPaginationResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Wheelingful.BLL.Models.Responses;
+
+public class FetchChapterPaginationResponse
```

Comment:
consider making it generic for all paged responses

Reply:
Good point. Will check the possibility.